rainier-tests/src/test/scala/com/stripe/rainier/core/ContinuousMixtureTest.scala

```diff
@@ -0,0 +1,241 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute.Evaluator
+import com.stripe.rainier.sampler._
+import org.scalatest.FunSuite
+
+class ContinuousMixtureTest extends FunSuite {
+  implicit val rng: RNG = ScalaRNG(1528673302081L)
+
+  def sbcCheck(description: String)(prior: Continuous): Unit = {
+    val emptyEvaluator = new Evaluator(Map.empty)
+
+    def binomialQuantile(q: Double, n: Int, p: Double): Int = {
```

Comment:
oh good, let's move this lower/upper quantile check inside SBC?

Reply:
I culted it from there! It's `private` inside SBC though, I'll unprivate it.